install/kubernetes/cilium/values.yaml.tmpl

```diff
@@ -411,6 +411,8 @@ bgp:
 bgpControlPlane:
   # -- Enables the BGP control plane.
   enabled: false
+  # -- Enable the BGPv2 APIs.
+  v2Enabled: false
```

Comment:
naming is hard, what do you think about just v2 ? 

Also we should think about deprecating timeline for v1 api as well.

Reply:
hmm, not entirely sure about just v2. `Enabled` suffix to v2 was inline with how other features are enabled/disabled.  After migration, this will get removed.  

Deprecating timelines are still be hazy - we are aiming to complete v2 development by v1.16 - deprecating probably a release later. ( don't quote me though on this ) :)